CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 * [ENHANCEMENT] Add `gen index` and `gen bloom` commands to tempo-cli. [#903](https://github.com/grafana/tempo/pull/903) (@annanay25)
 
 ## v1.1.0 / 2021-08-26
+* [CHANGE] Renamed CLI flag from `--storage.trace.maintenance-cycle` to `--storage.trace.blocklist_poll`. This is a **breaking change**  [#897](https://github.com/grafana/tempo/pull/897) (@mritunjaysharma394)
```

Comment:
one last change. we need to move this to the section above :point_up: . This change is not in 1.1. It is currently unreleased.

Reply:
oops, yes, I am just fixing it, thanks a lot!